src/Table/index.js

```diff
@@ -89,31 +91,32 @@ class Table extends PureComponent {
    * @return {jsx}
    */
   renderHeader() {
```

Comment:
Ok guys I need to know what do think about that. Should I let component as method here or put it in an external file `TableHeader`. Same question for the body and footer component.